src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String CORRECT_LOGIN = "GhostVoice";
+    private static final String CORRECT_PASSWORD = "123456";
+    private static final int CORRECT_AGE = 19;
+    private static final String SHORT_LOGIN = "Wally";
+    private static final String SHORT_PASSWORD = "Milky";
+    private static final int YOUNG_AGE = 12;
+
+    private RegistrationService registrator = new RegistrationServiceImpl();
+
+    @AfterEach
+    void tearDown() {
+        Storage.PEOPLE.clear();
+    }
+
```

Comment:
general recommendation regarding structuring your tests - try to group them:
1. successful tests
2. login tests (empty, null, longer, shorter, used etc.)
3. password tests
4. age tests

this way it will be easier for reader to find a corresponding test scenario